spec/code_objects/constants_spec.rb

```diff
@@ -66,7 +66,9 @@
   describe :BUILTIN_CLASSES do
     it "includes all base classes" do
       bad_names = []
-      YARD::CodeObjects::BUILTIN_CLASSES.each do |name|
+      builtin_classes = YARD::CodeObjects::BUILTIN_CLASSES
+      builtin_classes -= ["Bignum", "Fixnum"] if 0.class == Integer
```

Comment:
This change means we are no longer testing for all builtin classes. This should probably be wrapped in a version check, but at best, we should find a way to test this without emitting a warning, if possible.